### ISSUE TITLE
Normalize naming convention

### ISSUE THREAD

**Issue by dxcore35** [danielo515/obsidian-modal-form#242]
![Screenshot 2024-04-24 at 17 04 25](https://github.com/danielo515/obsidian-modal-form/assets/7279156/a53f99a2-8dcd-4d99-88bf-c3f6ce7bab93)

Please normalize these fields so it is clear what field is name (that is used in code) and what field  (that is used to display the field). Please keep this naming connection across the UI of the plugin, so there is consistency and friction free meaning of the field,  many times I flipped them....

**Comment by danielo515** [comment]
So you are suggesting:
- Form Name
- Form Label
- Field Name
- Field Label
Right?

**Comment by dxcore35** [comment]
Exactly! And then instead of explaining what is the function of the label and name by that long text you have in the form section. Maybe you can just add tooltip by hovering over the field it will show small description:

- Name - Variable name for the field (not shown, used when calling from code)
- Label - Text field name (displayed)

**Comment by danielo515** [comment]
For that I think it is better to:
- Have a compact view setting, so people can choose between compact and totally explained
- Have it default to show it on mobile? because hovering there doesn't work as good.

**Comment by dxcore35** [comment]
I think hovering over every field and get small text... that can be actually taken online form your documentation will be sufficient. I dont think anybody on mobile is going to build the forms :D But if you need you can leave the information emoticon after the field name, and when clicking on that they can get popup with description. I think this is overkill.
